lib/Exception/BaseRequestException.php

```diff
@@ -30,7 +30,7 @@ private function filterResponseForException($response)
     {
         try {
             $responseJson = $response->json();
-            $this->message = $responseJson["message"];
+            $this->message = $responseJson["error"] . ": " . $responseJson["error_description"];
```

Comment:
So I think we could add `error`, `error_description`, and also `errors` (that may come from some other module error payload) to the BaseRequestException class. This way, the dev is free to not only view the error but handle other kinds of errors at any convenience.

Reply:
Added these fields as properties of the class, and logic to set them in the constructor - additionally I changed the message to show the response body so any fields would be returned agnostic to their field name.